content/docs/how-we-work/baseline/_index.md

```diff
@@ -191,6 +191,81 @@ Follow these steps only if the project doesn't already exist.
+     * Save the rule to start receiving Slack notifications for the defined alerts.
+
+#### Creating Tickets from Alerts
+
+1. Review the Alert Details:
+   * Click on an alert to view detailed information, including stack traces, affected users, and the frequency of occurrences. Detailed error context helps in understanding the issue better and aids in faster resolution.
+   * **Error Context**: Review the contextual data provided by Sentry, such as the environment (production, staging), user actions leading to the error, and system state. This information is critical for diagnosing and resolving issues efficiently.
+
+2. Integrate Sentry with your ticketing system: (e.g. Jira, Github Issues).
+   * Navigate to **Project Settings** > **Integrations**.
+   * Select **Jira** and follow the prompts to authenticate and authorize Sentry to access your Jira instance.
+   * Configure the integration settings, such as default project and issue type, to streamline ticket creation.
+
+3. Create a Ticket:
+   * **Manually Creating a Ticket**:
+     * Click on the **Create Issue** button in the alert details view.
+     * Select **Jira** as the integration.
+     * Fill in the ticket details, including the summary, description, and priority. Include all relevant information provided by Sentry to ensure the issue is well-documented.
+     * Submit the ticket, which will then appear in your Jira project.
+   * **Automatically Creating Tickets**:
+     * Configure Sentry to automatically create Jira tickets for certain types of alerts. This can be done by setting up alert rules to trigger ticket creation.
+     * In **Project Settings** > **Alerts**, create a new alert rule and specify the conditions (e.g., error type, frequency) and actions (create a Jira ticket).
+     * Define the default project, issue type, and other ticket fields to ensure consistency and streamline the process.
+
+4. Managing Tickets in Jira:
+   * Once a ticket is created, ensure it is assigned to the appropriate team or individual. Jira’s workflow management tools can help track the progress of the ticket from creation to resolution.
+   * **Tracking and Reporting**: Use Jira’s reporting features to track the status of logged issues. Generate reports to identify trends, recurring issues, and areas for improvement.
+   * **Comments and Updates**: Utilize Jira’s commenting system to add updates, discuss potential solutions, and collaborate with team members directly within the ticket.
+
+### Best Practices
+
+* Ensure Comprehensive Logging:
+  * Ensure all critical parts of your Drupal application are covered by logging. This includes user interactions, database queries, API calls, and third-party integrations.
+  * **Error Levels**: Use appropriate error levels (debug, info, warning, error, critical) to categorize logs. This helps in filtering and prioritizing issues effectively.
```

Comment:
Sentry is not a good fit for `debug` and `info` logs. In fact, we should explicitly mention in set up instructions that they should not send debug and info logs. Even warning may be too much but that may be for the team to decide.